torch/distributed/_tensor/op_schema.py

```diff
@@ -131,7 +131,15 @@ def max_num_shards(self) -> int:
 
     @property
     def output_mesh_shape(self):
-        return self.strategies[0].output_spec.mesh.shape
+        output_spec = self.strategies[0].output_specs
+        if isinstance(output_spec, DTensorSpec):
+            return output_spec.mesh.shape
+        else:
+            assert isinstance(
+                output_spec, tuple
+            ), "found no DTensorSpec in the OpStrategy!"
+            assert output_spec[0] is not None
+            return output_spec[0].mesh.shape
```

Comment:
Just to confirm, when we have a tuple of output_spec, the mesh of each element in the output_spec would be identical. That's why we can simply return the first one.

Reply:
yes